pkg/bb/cmd/main.go

```diff
@@ -28,6 +28,14 @@ func init() {
 		if len(os.Args) == 0 {
 			log.Fatal("Arg len is 0. This is impossible")
 		}
+		// This is a gross hack for elvish.
+		// If you do this:
+		// elvish -c "echo hi" # works
+		// sh -c "echo hi" # fails b/c it does not parse it.
+		if len(os.Args) == 1 && !filepath.IsAbs(os.Args[0]) {
```

Comment:
Indeed this is gross. Doesn't this mean elvish will be executed for every relative command?

Reply:
"bc it does not parse it" what? This makes no actual sense

Reply:
Not having a shell or elvish is a valid configuration for bb, so this can't be a thing...

Reply:
it really, really, does not parse it. if you do this
sh -c "echo hi"
elvis will do an exec with this:
exec("echo hi", 0);
i.e. it will NOT parse the string, but instead use it as a name. 
it will NOT be executed for every relative command, since if a command is in PATH the shell will put a directory on the front. That's what path is for. If you do 
./command
however it will be. 

If there is no elvish, then this will fail. I can, further, see if there is an embedded space, or look for certain failures. I could even try to fix elvish but oh, that elvish code ...

This is the last thing I need to make chipsec work. I'm open to other ideas. I really hope there's a better way to do this.